cms/djangoapps/contentstore/views/component.py

```diff
@@ -469,7 +472,7 @@ def create_support_legend_dict():
             course_advanced_keys
         )
     if advanced_component_templates['templates']:
-        component_templates.insert(0, advanced_component_templates)
+        component_templates.append(advanced_component_templates)
```

Comment:
@navinkarkera What is the reason for this change?

Reply:
To move `Advanced` button to end as requested [here](https://github.com/openedx/frontend-app-authoring/issues/1462#issuecomment-2465876156).